scheduler/src/cook/scheduler/scheduler.clj

```diff
@@ -653,7 +654,8 @@
             (not (and is-rate-limited? enforcing-job-launch-rate-limit?))))
         considerable-jobs
         (->> pending-jobs
-             (tools/filter-based-on-quota user->quota user->usage)
+             ;(tools/filter-based-on-pool-quota (tools/get-quota-for-pool (config/pool-quotas) pool-name))
+             (tools/filter-pending-jobs-for-autoscaling user->quota user->usage (tools/get-quota-for-pool (config/pool-quotas) pool-name))
```

Comment:
It seems odd to me to use a function named `for-autoscaling` here, when this has nothing to do with autoscaling. Maybe the function this calls should just be named `filter-based-on-quota`, and that would do both the user-level and the global filtering?

Reply:
Works for me.